server/models/documents.js

```diff
@@ -0,0 +1,35 @@
+export default (sequelize, DataTypes) => {
+  const Documents = sequelize.define('Documents', {
```

Comment:
You should add validations to your models as well.

Relevant link: http://docs.sequelizejs.com/manual/tutorial/models-definition.html#validations